x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/tmp_routes.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
you can import `epmRouteService` (already in `/common`!) into [services](https://github.com/elastic/kibana/blob/feature-ingest/x-pack/plugins/ingest_manager/public/applications/ingest_manager/services/index.ts) to replace this